### ISSUE TITLE
Use it on Intune

### ISSUE THREAD

**Issue by kkatsariotis** [sgmills/PrivilegesDemoter#12]
What it needs to use it on Intune (no Jamf, no IBM notifier in place)?

**Comment by salihzett** [comment]
Did u find a solution for your case? 
my idea ist to install IBM notifier or use another notifier.

**Comment by kkatsariotis** [comment]
> Did u find a solution for your case? my idea ist to install IBM notifier or use another notifier.

No. I tried with a script running every 30-60 mins from Intune and demoting the privileged account. But the IT sec department preferred a commercial solution, with approvals on every request, etc

**Comment by sgmills** [comment]
It should be easier to deploy on Intune with version 3, which is currently in pre-release.

The new version no longer relies on Jamf to run a policy. Instead, it is all self contained in a local script, you just have to deploy a notification agent such as IBM Notifier or Swift Dialog.
